src/components/Notification/Notification.tsx

```diff
@@ -26,7 +26,7 @@ export type Props = {
   /**
    * A list of up to two actions that the notification can perform.
    */
-  actions?: readonly [NotificationAction?, NotificationAction?];
+  actions?: NotificationAction[];
```

Comment:
Restricting the number of actions to 2 was causing some issues in places where we wanted to push to an existing array, e.g.
``` typescript
const actions: NotificationProps["actions"] = [action];
if (condition) {
  actions.push(extraAction); // "push" does not exist on [NotificationAction?, NotificationAction?]
}
```

Similarly, with concat:
``` typescript
let actions: NotificationProps["actions"] = [action];
if (condition) {
  actions = actions.concat([extraAction]); // source could have length greater than 2
}
```

So to keep it simple I've removed the length restriction.